pyspark_ai/pyspark_ai.py

```diff
@@ -99,6 +104,7 @@ def __init__(
             ).search
         else:
             self._cache = None
+        self._vector_store_dir = vector_store_dir
```

Comment:
We need to have a default value for the `vector_store_dir`. For example, "spark_ai_vector_store" under the temp dir of the file system.

Reply:
Oh...seems the default is in-memory